src/design-tokens/ThemeProvider.tsx

```diff
@@ -1,13 +1,27 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { ThemeProvider as MuiThemeProvider } from '@material-ui/core/styles';
 import { ThemeProvider as EmotionThemeProvider } from 'emotion-theming';
 
-import theme from './theme';
+import ThemeContext from './ThemeContext';
+import { getTheme, ThemeMode } from './theme';
 
-const ThemeProvider: React.FC = ({ children }) => (
-  <EmotionThemeProvider theme={theme}>
-    <MuiThemeProvider theme={theme}>{children}</MuiThemeProvider>
-  </EmotionThemeProvider>
-);
+const ThemeProvider: React.FC = ({ children }) => {
+  const isDarkModeDefault = window?.__VERDACCIO_BASENAME_UI_OPTIONS?.dark_mode;
+  const [isDarkMode, setIsDarkMode] = useState(!!isDarkModeDefault);
```

Comment:
This works really well, but we might use the local storage to persist in the dark mode. Let's imagine the following scenario:
* Admin set `dark_mode` as `true` 
* Dark mode is set by default for all users.
* But here is one which likes green or primary color.
* This user will try to persist primary theme but it is not possible right now, on refresh it will switch again.

Reply:
Yes I noticed this problem and I was thinking to add session storage or local storage, but maybe I could do i in another PR?